source/develop/release-management/features/virt/enhance-import-export-with-ova.md

```diff
@@ -243,18 +228,16 @@ theory, it should look like this:
 ![](../../../../images/wiki/download-ova.png)
 
 1. The front end passes the back end a VM (or template) to download.
-2. The back end sends import parameters to imageio-proxy in order to
-   prepare to download-ova.
-3. The URL that is returned by imageio-proxy is returned to the front
-   end through the back end.
-4. The front end initializes a target file to write the OVA to, and
-   starts listening on the URL.
-5. The back end passes the disk to imageio-proxy to copy, and
-   imageio-proxy delivers it to imageio.
-6. Imageio starts reading the disk - converting it to QCOW - and sends
-   it to the URL. The front end then streams the data into the target
-   file.
-7. The back end sends the OVF configuation to imageio-proxy.
-   Imageio-proxy sends it to imageio, and imageio writes the OVF
-   configuration to the URL. The front end then streams the data into
-   the target file.
+2. The back end starts the exporting to OVA flow (as described in 
+   'Exporting a VM or Template as an OVA')
+3. The front end displays a link for downloading the OVA.
+4. Press on the download link will start an ovirt-imageio download
+   to the browser (not implemented yet in the front end).
+
+Notes:
+
+1. If possible, we'll consider using streaming the disk data directly
+   from the qemu chain to the browser.
+2. The flow should be supported in the sdk and future automation using
+   Ansible.
```

Comment:
some would say that we should even start with the sdk since it is more important that integrating it with the current UI, but ok..